src/Blockcore.Generator/Program.cs

```diff
@@ -1,78 +1,88 @@
 using System;
 using System.Collections.Generic;
+using System.Text.Json;
 using NBitcoin;
 
 namespace Blockcore.Generator
 {
-    class Program
+   class GeneratorOutput
```

Comment:
This could probably be moved into separate file, together with the "GenesisBlock" type as well. But that's something for the future!